gradle-plugins/model-compiler/src/test/java/io/spine/gradle/compiler/annotation/Given.java

```diff
@@ -48,7 +44,7 @@
 /**
  * @author Dmytro Grankin
  */
-public class Given {
+class Given {
```

Comment:
In Spine core we put `Given`s to a separate package (have a look) to avoid using an internal API. Please try to do the same in this project. At least for the `Given`s you modify.